boards/com.matternet.bmu_4.3/board.ld

```diff
@@ -0,0 +1,114 @@
+MEMORY
+{
+    bl (rx) :             ORIGIN = 0x08000000,            LENGTH = 12K
+    app (rx) :            ORIGIN = 0x08000000+12K,        LENGTH = 128K-12K-2K
+    params (rx) :         ORIGIN = 0x08000000+126K,        LENGTH = 2K
```

Comment:
could you please align this LENGTH column with the others?

Reply:
Done!